packages/dataviews/src/dataviews.js

```diff
@@ -68,12 +68,14 @@ export default function DataViews( {
 							onChangeView={ onChangeView }
 						/>
 					</HStack>
-					<ViewActions
-						fields={ fields }
-						view={ view }
-						onChangeView={ onChangeView }
-						supportedLayouts={ supportedLayouts }
-					/>
+					{ window?.__experimentalAdminViews && (
```

Comment:
We should probably just hide the `ViewTypeMenu`, no?

Reply:
I'm open to that.